debian/libmiral3.symbols

```diff
@@ -297,3 +297,4 @@ libmiral.so.3 libmiral3 #MINVER#
  (c++)"vtable for miral::WindowManagementPolicy@MIRAL_2.0" 2.0.0
  MIRAL_2.1@MIRAL_2.1 2.1.0
  (c++)"miral::CanonicalWindowManagerPolicy::confirm_placement_on_display(miral::WindowInfo const&, MirWindowState, mir::geometry::Rectangle const&)@MIRAL_2.1" 2.1.0
+ (c++)"miral::CanonicalWindowManagerPolicy::handle_request_drag_and_drop(miral::WindowInfo&)@MIRAL_2.1" 2.1.0
```

Comment:
I had to check that we'll have the right libmiral3 version, here, but we will; yay!

bors r+